tools/grab_micro_posts.py

```
@@ -62,15 +71,15 @@
                         post = frontmatter.load(f)
                         if 'note_id' in post.metadata:
                             current_id = post.metadata['note_id']
-                            print(f"Found note_id: {current_id} in file: {file_path}")
+                            logging.info(f"Found note_id: {current_id} in file: {file_path}")
                             if isinstance(current_id, int):
                                 highest_note_id = max(highest_note_id, current_id)
                             else:
-                                print(f"Warning: non-integer note_id found: {current_id} in file: {file_path}")
+                                logging.warning(f"Non-integer note_id found: {current_id} in file: {file_path}")
                 except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)